libc-test/build.rs

```diff
@@ -192,6 +199,9 @@ fn main() {
             // OSX calls this something else
             "sighandler_t" if bsdlike => "sig_t".to_string(),
 
+            // does not exist on DragonFly
+            "fflags_t" if dragonfly => "uint32_t".to_string(),
```

Comment:
It's probably best if the dragonfly bindings avoid `fflags_t` entirely if it doesn't exist on dragonfly